README.md

```diff
@@ -51,6 +51,7 @@ Choose a language:
 * Using the CDK for Terraform [escape hatch](./docs/working-with-cdk-for-terraform/escape-hatch.md).
 * Using the CDK for Terraform [tokens](./docs/working-with-cdk-for-terraform/tokens.md).
 * Using Terraform [data sources](./docs/working-with-cdk-for-terraform/data-sources.md).
+* Using Terraform [remote state](./docs/working-with-cdk-for-terraform/remote-state.md).
```

Comment:
Since `remote state` is effectively a data source, let's move that to a section in the `data source` docs.

Reply:
We can call it "Terraform Remote State".